ProcessLib/HeatTransportBHE/LocalAssemblers/HeatTransportBHELocalAssemblerSoil-impl.h

```diff
@@ -139,7 +139,7 @@ void HeatTransportBHELocalAssemblerSoil<ShapeFunction, IntegrationMethod>::
                 .property(MaterialPropertyLib::PropertyType::phase_velocity)
                 .value(vars, pos, t);
         auto const velocity = Eigen::Map<Eigen::Vector3d const>(
-            std::get<MaterialPropertyLib::Vector>(velocity_arr).data(), 1, 3);
+            std::get<MaterialPropertyLib::Vector>(velocity_arr).data(), 3, 1);
```

Comment:
I'd think you don't need to specify any of the size arguments in this case.
Could you try it w/o size arguments, please?

Reply:
You are right. Changed.